auth-persistence/src/main/java/org/iqu/auth/persistence/dao/DaoManager.java

```diff
@@ -0,0 +1,51 @@
+package org.iqu.auth.persistence.dao;
+
+import static org.iqu.auth.persistence.dao.JdbcConstants.*;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.apache.commons.dbcp.BasicDataSource;
+
+/**
+ * Manages database connection.
+ * 
+ * @author Mitroi Stefan-Daniel
+ *
+ */
+public class DaoManager {
+
+	private static DaoManager instance = new DaoManager();
+	private Connection connection;
+
+	public static DaoManager getInstace() {
+		return instance;
+	}
+
+	public Connection getConnection() {
+
+		BasicDataSource dataSource = new BasicDataSource();
+		dataSource.setDriverClassName(DRIVER);
+		dataSource.setUrl(URL);
+		dataSource.setUsername(USERNAME);
+		dataSource.setPassword(PASSWORD);
+		try {
+			connection = dataSource.getConnection();
+		} catch (SQLException e) {
+			e.printStackTrace();
```

Comment:
Scri in log nu printezi stacktrace. Daca nu poti sa stabilesti conexiunea, care ar fi urmatorul pas?